xonsh/proc.py

```diff
@@ -336,9 +342,67 @@ def wrapped_simple_command(args, stdin, stdout, stderr):
                         stderr.write(r[1])
                 elif r is not None:
                     stdout.write(str(r))
-                return True
+                return 0  # returncode for succees
```

Comment:
Hmm, yeah, I understand. They definitely got this backwards with `0` being True.  On the otherhand, it is such a strong part standard/convention/whatever that it is probably worth it to not break this. For the `and/or` stuff, testing `rtn == 0` probably isn't the worst thing in the world, though it is annoying :)